constraints.pro

```diff
@@ -155,6 +155,10 @@ gen_enforced_field(WorkspaceCwd, 'repository.type', 'git').
 % packages.
 gen_enforced_field(WorkspaceCwd, 'repository.url', 'https://github.com/MetaMask/controllers.git').
 
+% "bugs.url" must be "https://github.com/MetaMask/controllers/issues" for all
```

Comment:
For most of the constraints here I opted to apply the constraint just to public packages and leave the root package out. Considering that the root package is private should we do that here too? Or how do we expect this field to be used?

Reply:
I was just following the same pattern as for the `repository` field. I don't expect the field to be used, I can update this to exclude private packages.

Reply:
Addressed here: https://github.com/MetaMask/controllers/pull/1030/commits/9abd9b8badc570c136f1aa0434169dc9653c83b8

Reply:
Cool. This PR reminded me that I made some changes to the constraints file when I created the monorepo template to remove references to the `controllers` repo. They require that the root package have a repository URL, so your change should fit nicely with them.